docs/reference/indices/data-stream-stats.asciidoc

```diff
@@ -0,0 +1,186 @@
+[role="xpack"]
+[[data-stream-stats-api]]
+=== Data stream stats API
+++++
+<titleabbrev>Data stream stats</titleabbrev>
+++++
+
+experimental::[]
+
+Retrieves statistics for one or more <<data-streams,data streams>>.
+
+////
+[source,console]
+----
+PUT /_index_template/template
+{
+  "index_patterns": ["my-data-stream*"],
+  "data_stream": { }
+}
+
+PUT /my-data-stream/_bulk?refresh
+{"create":{ }}
+{ "@timestamp": "2020-12-08T11:04:05.000Z" }
+{"create":{ }}
+{ "@timestamp": "2020-12-08T11:06:07.000Z" }
+{"create":{ }}
+{ "@timestamp": "2020-12-09T11:07:08.000Z" }
+
+POST /my-data-stream/_rollover/
+POST /my-data-stream/_rollover/
+
+PUT /my-data-stream-two/_bulk?refresh
+{"create":{ }}
+{ "@timestamp": "2020-12-08T11:04:05.000Z" }
+{"create":{ }}
+{ "@timestamp": "2020-12-08T11:06:07.000Z" }
+
+POST /my-data-stream-two/_rollover/
+----
+// TESTSETUP
+////
+
+////
+[source,console]
+----
+DELETE /_data_stream/*
+DELETE /_index_template/*
+----
+// TEARDOWN
+////
+
+[source,console]
+----
+GET /_data_stream/my-data-stream/_stats
+----
+
+
+[[data-stream-stats-api-request]]
+==== {api-request-title}
+
+`GET /_data_stream/<data-stream>`
+
+
+[[data-stream-stats-api-path-params]]
+==== {api-path-parms-title}
+
+`<data-stream>`::
+(Optional, string)
+Comma-separated list of data streams used to limit the request. Wildcard
+expressions (`*`) are supported.
++
+To target all data streams in a cluster, omit this parameter or use `*`.
+
+
+[role="child_attributes"]
+[[data-stream-stats-api-response-body]]
+==== {api-response-body-title}
+
+`_shards`::
+(object)
+Contains information about shards that attempted to execute the request.
++
+.Properties of `_shards`
+[%collapsible%open]
+====
+`total`::
+(integer)
+Total number of shards that attempted to execute the request.
+
+`successful`::
+(integer)
+Number of shards that successfully executed the request.
+
+`failed`::
+(integer)
+Number of shards that failed to execute the request.
+====
+
+`data_stream_count`::
+(integer)
+Total number of selected data streams.
+
+`backing_indices`::
+(integer)
+Total number of backing indices for the selected data streams.
+
+`total_store_size_bytes`::
+(integer)
+Total size, in bytes, of all shards for the selected data streams.
+
+`data_streams`::
+(array of objects)
+Contains statistics for the selected data streams.
++
+.Properties of objects in `data_streams`
+[%collapsible%open]
+====
+`data_stream`::
+(string)
+Name of the data stream.
+
+`backing_indices`::
+(integer)
+Current number of backing indices for the data stream.
+
+`store_size_bytes`::
```

Comment:
we also have human readable versions of this field and `total_store_size`, if we wanted to document those.